### ISSUE TITLE
Is the requirement "azure==4.0.0" excessive?

### ISSUE THREAD

**Issue by snorrea** [scottwernervt/cloudstorage#79]
AFAIK this library is an abstraction over Azure Storage. If so, why does it require the azure module? In practice, this installs the full suite of Azure modules, including modules required to manage CosmosDB, Azure Batch, and even Azure Compute. I'm worried that this requirement bloats any environments installing this module for use with Azure, and necessitates other dependencies due to all the other modules that get installed.

Am I missing something?

**Comment by scottwernervt** [comment]
> Am I missing something?

No. I think the older azure package was required for azure storage access. #71 will switch to the latest azure storage package and eliminate this.
